libs/ett/shared/util-browser/src/lib/eo-title.service.ts

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { Injectable } from '@angular/core';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { Title } from '@angular/platform-browser';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class EoTitle {
+  private titleSubject = new BehaviorSubject<string>('');
+  title$!: Observable<string>;
```

Comment:
issue: We don't use non-null assertions. We should have/find a lint rule for this 🤔